florestad/src/florestad.rs

```diff
@@ -426,7 +427,7 @@ impl Florestad {
             match self.create_tls_config(&data_dir) {
                 Ok(config) => Some(config),
                 Err(_) => {
-                    error!("Failed to load SSL certificates, ignoring SSL");
+                    warn!("Failed to load SSL certificates, ignoring SSL");
```

Comment:
Could you do the same to the config not found warning ?

Reply:
Agree

Reply:
Just got to it. But I also changed the message to make it clear it isn't an error. Or should I keep it as before just with the warn flag?

Reply:
For me [9b53417](https://github.com/vinteumorg/Floresta/pull/265/commits/9b53417f2f40a0c628406f881b8157e96e150bab) is fine.